### ISSUE TITLE
Run two UI instances of Pokemate

### ISSUE THREAD

**Issue by gerardcuadras-zz** [SwipeX/PokeMate#270]
When trying to run two accounts I get this error on the second one:

`Error: User data directory [null (/Users/gerardcuadras/Library/Application Support/dekk.pw.pokemate.PokeMateUI/webview)] is already in use`

The second instance has the same walking path as the first one.

**Comment by giubil** [comment]
Try starting it with `show=false` in the settings and see if that helps

**Comment by gerardcuadras-zz** [comment]
Yes, now it's working. But, I will prefer to run with `show=true`. 😁

**Comment by maxi2mc** [comment]
It works for me, but I have two different folders under Windows 10 using latest java JRE.

**Comment by gerardcuadras-zz** [comment]
I have two different folders too (Mac), but I get that error.😕

**Comment by SippieCup** [comment]
Seems like an issue with javafx. Don't know if we will be doing anything about it for now. I know that I personally can't duplicate this on my mac.

**Comment by SwipeX** [comment]
looks like the way mac implemented the webview in javaFX. Something we hope to move away from.
